python-packages/contract_wrappers/src/zero_ex/contract_wrappers/tx_params.py

```diff
@@ -0,0 +1,37 @@
+"""Transaction Parameters class."""
+from typing import Optional
+import attr
```

Comment:
Add a blank line here, to separate standard library imports from 3rd party imports.

Reply:
done.